README.md

```diff
@@ -1,6 +1,28 @@
 # kartograafr
 ArcGIS/Canvas data bridge
 
+# NOTES:
+Probably need to install Anaconda as separate environment to run
+native on OSX and to run inside eclipse.  See https://docs.anaconda.com/anaconda/user-guide/tasks/integration/eclipse-pydev
```

Comment:
Probably some more insight in README is needed why Anaconda distribution is needed to run the Application? Also, i am curious why is ArcGIS needs it for python3 version

Reply:
The README will be fixed in another commit.

ESRI just doesn't support using pip to install it for reasons of their own.  I suspect it requires other code that Anaconda supports and pip doesn't.